astropy/cosmology/core.py

```diff
@@ -1825,7 +1825,7 @@ def efunc(self, z):
         return np.sqrt(zp1 ** 3 * (Or * zp1 + Om0) + Ode0)
 
     def inv_efunc(self, z):
-        """Function used to calculate :math:`\frac{1}{H_z}`.
+        """Function used to calculate :math:`\\frac{1}{H_z}`.
```

Comment:
So this is the only one that need to be changed.  I'll update this PR.  I guess your preference here is to simply prepend a "r" instead?